circle.yml

```diff
@@ -26,6 +26,8 @@ jobs:
       - run:
           command:
             bash exec/prep-key.sh
+      - run:
+          command:
             chmod +x exec/apk-gen.sh
             ./exec/apk-gen.sh
```

Comment:
Not this. `bash exec/apk-gen.sh`

Reply:
@iamareebjamal I combined it in the second commit but the CI showed an error of permission denied